__test__/op-connect.test.ts

```diff
@@ -169,6 +158,94 @@ describe("Test OnePasswordConnect CRUD", () => {
         expect(vaults[0].name).toEqual(title);
         expect(vaults[1].name).toEqual(title);
     });
+
+    describe("getVaultByTitle", () => {
+
+        test("should throw an error if no vaults found", async () => {
+            const op = OnePasswordConnect(testOpts);
+            const title = "awsome_title";
+
+            nock(mockServerUrl)
+                .get("/v1/vaults")
+                .query({
+                    filter: `title eq "${title}"`,
+                })
+                .reply(200, []);
+
+            await expect(() => op.getVaultByTitle(title)).rejects.toEqual(HttpErrorFactory.noVaultsFoundByTitle());
+        });
+
+        test("should throw an error if more than 1 vault found", async () => {
+            const op = OnePasswordConnect(testOpts);
+            const title = "awsome_title";
+
+            nock(mockServerUrl)
+                .get("/v1/vaults")
+                .query({
+                    filter: `title eq "${title}"`,
+                })
+                .reply(200, [{}, {}]);
+
+            await expect(() => op.getVaultByTitle(title)).rejects.toEqual(HttpErrorFactory.multipleVaultsFoundByTitle());
+        });
+
+        test("should return vault", async () => {
+            const op = OnePasswordConnect(testOpts);
+            const title = "awsome_title";
+
+            nock(mockServerUrl)
+                .get("/v1/vaults")
+                .query({
+                    filter: `title eq "${title}"`,
+                })
+                .reply(200, [{ name: title } as Vault]);
+
+            const vault: Vault = await op.getVaultByTitle(title);
+
+            expect(vault.name).toEqual(title);
+        });
+
+    });
+
+    describe("getVault", () => {
```

Comment:
Tiny nitpick to be consistent with other test descriptions:
```suggestion
    describe("get vault", () => {
```

Reply:
fixed